sqlitecluster/SQLitePool.cpp

```diff
@@ -60,7 +60,7 @@ size_t SQLitePool::getIndex(bool createHandle) {
             return index;
         } else {
             // Wait for a handle.
-            SINFO("Waiting for DB handle");
+            SWARN("Waiting for DB handle");
```

Comment:
Won't this one specifically be written in a loop while the server is receiving requests? not sure if that's a problem though, just noting.

Reply:
It's just as much of a problem when it's an `info` 😄

Reply:
I know; I meant, do we want to continue logging them all the time?

And what will happen if instead of thousands of lines with info we do thousands with warn?

Reply:
> And what will happen if instead of thousands of lines with info we do thousands with warn?

I think we get a bugbot issue created but that's about it.

Reply:
+1